core/templates/dev/head/pages/exploration_editor/editor_tab/StateInteraction.js

```diff
@@ -244,6 +244,21 @@ oppia.controller('StateInteraction', [
                   stateCustomizationArgsService.displayed).length > 0);
               }
 
+              $scope.areCustomizationArgsValid = function() {
+                if (!stateCustomizationArgsService.displayed) {
+                  return false;
+                }
+                var validationService =
+                  $injector.get(
+                    INTERACTION_SPECS[
+                      $scope.stateInteractionIdService.displayed].id +
+                    'ValidationService');
+                var warningsList =
+                  validationService.getCustomizationArgsWarnings(
+                    stateCustomizationArgsService.displayed);
+                return warningsList.length == 0;
```

Comment:
@seanlip Ok, I assume it's a rule in Oppia to always use === and !== for everything? I don't think in this case there could be any type conversion problem, since we know the types of both sides of the condition. But if it's a rule, I'll follow it.